docs/release.md

```diff
@@ -39,28 +39,45 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This release cycle mirrors the Kubernetes' and allows KubeVirt to release after the community.
+This looks like:
+| Week Number in Year | K8s Release Number | K8s Release Week | KubeVirt Release Week |                     Note                    |
+|:-------------------:|:------------------:|:----------------:|-----------------------|:-------------------------------------------:|
+| 1                   | 1.24               | 1 (January 03)   | 1 (~January 03)       |                                             |
+| 15                  | 1.24               | 15 (April 12)    | 15 (~April 12)        |                                             |
```

Comment:
1.24 is listed for both the 1st and 15th week in this table.  Maybe we can have some clarification here on what each of these dates means.

Are you saying that on the 1st week 1.24 release branch is created? and the 15th week is the GA release of 1.24?

Reply:
Is this maybe -rc reference?